src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-duration: 300ms;
+
+    &:hover {
+      transform: scale(1.2, 1.2);
+      color: #34568b;
+    }
```

Comment:
It's better to specify what properties you are animate https://mateacademy.slack.com/archives/C02LA8ED589/p1638185091084600

Reply:
fixed & upd